### PR TITLE
Add Banc Sabadell

### PR DESCRIPTION
**Pull request by VictorBjelkholm**
One of the biggest banks in Spain only allow you to have 4 numbers as a password.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,9 @@ Financial services - where we don't allow you to create the strongest password p
 We put punctuation in the list of unusable characters to make it additionally confusing.
 
 ![Williams-Sonoma](/screenshots/williams-sonoma.png "Williams-Sonoma")
+
+### [Banc Sabadell](https://www.bancsabadell.com)
+
+One of the biggest spanish banks allows you to only use numbers and just four of them.
```
Comment (AdamBunn) [inline README.md]: Is this the card pin, the account password, or both?

Comment (duffn) [inline README.md]: Yes, this looks like a PIN, which doesn't seem completely unreasonable to me.

**Comment by duffn** [comment]
Closing as I don't see this one fitting the bill as it appears to be a PIN. Allowing only numbers in tnis instance doesn't seem unreasonable to me.

**Comment by VictorBjelkholm** [comment]
No, this the is the access code for the online portal, where you make
transfers and get account status and such. PIN change has to happen at a
ATM. Maybe my wording in the description was a bit off...

**Comment by duffn** [comment]
Got it - that's definitely a dumb password rule then. I think this is worth adding.

**Pull request closed without merging**